<map id="exception::exception" name="exception::exception">
<area shape="rect" id="node1" title=" " alt="" coords="5,31,148,57"/>
<area shape="rect" id="node2" href="$group__python__builtins.html#gacc0a69f008d341c07ff939e0580c5985" title=" " alt="" coords="213,5,276,32"/>
<area shape="rect" id="node4" href="$detail_2common_8h.html#aeea9746c2787bab3a5a0c983ef965520" title="Used internally" alt="" coords="196,56,293,83"/>
<area shape="rect" id="node3" href="$classhandle.html#aa58c40063166c7cedefdc4db2d6fa608" title="Return the underlying PyObject * pointer" alt="" coords="341,5,424,32"/>
</map>
